myp5.js

```diff
@@ -51,44 +47,53 @@ function draw() {
   }  
 }
 
-function lyrics(x, y) {
+function loadLyricsOnce() {
+
+  for (var i = 0; i < lyrics.length; i++) {
+    lyricObjects[i] = new Lyric(random(width), random(height), lyrics[i]);
+  }
+
+}
+
+function Lyric(x, y, _text) {
   this.acceleration = createVector(0, 0);
   this.velocity = p5.Vector.random2D();
   this.position = createVector(x, y);
   this.r = 3.0;
   this.maxspeed = 3;    // Maximum speed
   this.maxforce = 0.05; // Maximum steering force
+  this.text = _text;
 }
 
-lyrics.prototype.run = function(lyrics) {
-  this.flock(lyrics);
+Lyric.prototype.run = function() {
+ // this.flock(lyrics);
   this.update();
```

Comment:
Lets turn off flocking for now, thats a little too complicated and we're not setup properly for it